x/oracle/simulations/operations.go

```diff
@@ -26,6 +26,9 @@ const (
 	OpWeightMsgAggregateExchangeRateVote    = "op_weight_msg_exchange_rate_aggregate_vote"    //nolint: gosec
 	OpWeightMsgDelegateFeedConsent          = "op_weight_msg_exchange_feed_consent"           //nolint: gosec
 
+	DefaultWeightMsgSend               = 100 // from simappparams.DefaultWeightMsgSend
```

Comment:
If this const is used here and also for airdrop let's put it in a central location

Reply:
Moved them to ojo/util/sim/params.go 👍